### DISCUSSION TITLE
How to enable Bitbucket in open source version of middleware

### DISCUSSION THREAD

**Discussion by dicksonchellakon** [middlewarehq/middleware#452] [General]
We are exploring the open-source version of middleware with bitbucket, but it only integrates Github.  Is there any config to enable it? or is the Bitbucket supported in the Paid version?

**Comment by jayantbh** [comment]
Hey @dicksonchellakon,

Currently only Github is supported, and Gitlab is coming soon.
Bitbucket is ahead in the roadmap as well, due in a couple of months. :)

It might be worth trying out the trial (no card needed) of the cloud version in the mean-time.

**Comment by jayantbh** [comment]
Hey everyone who is interested in this feature 👋🏽, **Gitlab** is live. **Bitbucket** is planned in the near future. :)
Please upvote or react to the discussion thread if this is something you'd like.

OR...

### Vote here! https://github.com/middlewarehq/middleware/discussions/464

**Comment by abhi-atpl** [comment]
I want to integrate middleware with gitea. Can anyone one help me on this !!?
